### PR TITLE
Fix SELinux troubleshooting styling issues

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

**Comment by dperpeet** [comment]
Looks good, but there are a few decisions that deviate from what @andreasn decided.

Most noticeable are the changed "Enforce policy" and the fact that the "apply" buttons are now at the bottom for each "fix" item. Originally we moved the button up, because the fix-text for a single entry can be very, very long (when it lists all possible options).

screenshot from 2016-05-31 12-01-02

**Comment by andreasn** [comment]
I like the cleanup of the lines in the Solutions.

My mockup has no header https://raw.githubusercontent.com/cockpit-project/cockpit-design/master/selinux-troubleshooting/selinux-v5.png for the On/Off switch label, but for clarity, I would be OK with adding one, since it puts it more on an even playing field with the access control errors below.

I agree with @dperpeet with regards to the placing of the "Apply this solution" buttons. It's supposed to line up in the middle.

**Comment by stefwalter** [comment]
> I agree with @dperpeet with regards to the placing of the "Apply this solution" buttons. It's supposed to line up in the middle.

Have you tried this out? As is, looks really strange, with arbitrary wrapping in strange places in sentences. I agree that the designs look good, but the implementations looks nothing like the designs, not the least of which is that the listing view has completely changed.

**Comment by stefwalter** [comment]
screenshot from 2016-05-31 13-55-31

**Comment by andreasn** [comment]
Updated mockup:
selinux-v6

**Comment by andreasn** [comment]
The popup wording is from https://github.com/cockpit-project/cockpit/issues/4509#issuecomment-222666721

**Comment by dperpeet** [comment]
I can agree with moving the button further down, but the current proposal conflicts with the status alert once a fix has been applied:
screenshot from 2016-05-31 14-14-27

**Comment by dperpeet** [comment]
Adding this to setroubleshoot.css fixed the issue:
```
.list-group-item .alert {
    margin-top: 10px;
}
```

**Comment by dperpeet** [comment]
I think these commits should be squashed somewhat - e.g. combine the styling fixes.

**Comment by stefwalter** [comment]
> I think these commits should be squashed somewhat - e.g. combine the styling fixes.

Really? I was thinking that that it helped figure out what's what. Each change in independent, and could be referenced elsewhere, copied or consolidated later.

I'd be happy to squash the change you note above in though.

**Comment by stefwalter** [comment]
> Adding this to setroubleshoot.css fixed the issue:

Squashed that into the first commit.

File: pkg/selinux/setroubleshoot.css [hunk 1]
```diff
@@ -74,18 +70,42 @@ div.alert-ct-top {
...
-    font-size: 22px;
-    padding: 10px 0px 10px;
-    font-weight: 300;
-    margin-top: 0px;
+.list-group-item .alert {
+    margin-top: 10px;
+}
+
+.selinux-policy-ct label {
+    padding-left: 45px;
```
Comment (andreasn) [inline pkg/selinux/setroubleshoot.css]: I think it looks better without this.

**Pull request closed without merging**